tests/test_file.py

```diff
@@ -241,179 +238,12 @@ def test_access_token_credentials(self):
 
         self.assertNotEquals(None, credentials)
         self.assertEquals('foo', credentials.access_token)
-        mode = os.stat(FILENAME).st_mode
+        os.stat(FILENAME).st_mode
```

Comment:
Yeah, I was just trying to keep my linter from yelling at me. Fixed to your suggestion.

(Also, I try to avoid doing much more than that only legacy code unless it's my intention. Sort of like not boiling the ocean, but I'll gladly apply heat when you want).